src/app/util/ember-compatibility-functions.cpp

```diff
@@ -248,6 +248,34 @@ bool ServerClusterCommandExists(const ConcreteCommandPath & aCommandPath)
 
 namespace {
 
+CHIP_ERROR ReadClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId, DataVersion & aDataVersion)
+{
+    EmberAfCluster * cluster = emberAfFindCluster(aEndpointId, aClusterId, CLUSTER_MASK_SERVER);
+    if (cluster == nullptr)
+    {
+        return CHIP_ERROR_NOT_FOUND;
+    }
+    aDataVersion = *(cluster->version);
+    return CHIP_NO_ERROR;
+}
+
+void IncreaseClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId)
```

Comment:
Again, pass `ConcreteClusterPath`?

Reply:
have removed ConcreteClusterPath, it seems there is no need to use that one.